pipelines/protocolB.yaml

```diff
@@ -0,0 +1,278 @@
+    class: lsst.cp.pipe.defects.MeasureDefectsTask
+    config:
+      connections.inputExp: 'cptBrightDefectsProc'
+      connections.outputDefects: 'cptPartialBrightDefects'
+  cptMergeBrightDefects:
+    class: lsst.cp.pipe.defects.MergeDefectsTask
+    config:
+      connections.inputDefects: 'cptPartialBrightDefects'
+      connections.mergedDefects: 'brightDefects'
+
+  #low-value (dark pixels) defects
+  cptDefectIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'flat'
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cptDefectsProc'
+      connections.bias: 'bias'
+      connections.dark: 'dark'
+      connections.flat: 'flat'
+      doWrite: true
+      doOverscan: true
+      doAssembleCcd: true
+      doBias: true
+      doVariance: false
+      doLinearize: parameters.linearize
+      doCrosstalk: parameters.crosstalk
+      doBrighterFatter: false
+      doDark: false
+      doStrayLight: false
+      doFlat: false
+      doFringe: false
+      doApplyGains: false
+      doDefect: parameters.defects
+      doSaturationInterpolation: false
+      growSaturationFootprintSize: 0
+  cptMeasureDefects:
+    class: lsst.cp.pipe.defects.MeasureDefectsTask
+    config:
+      connections.inputExp: 'cptDefectsProc'
+      connections.outputDefects: 'cptPartialDefects'
+  cptMergeDefects:
+    class: lsst.cp.pipe.defects.MergeDefectsTask
+    config:
+      connections.inputDefects: 'cptPartialDefects'
+      connections.mergedDefects: 'faintDefects'
+
+  # # PTC:
+  cptPtcIsr:
+    class: lsst.cp.testing.CptIsrTask
+    config:
+      expectedExposureType: 'flat'
+      connections.ccdExposure: 'raw'
+      connections.outputExposure: 'cptPtcProc'
+      connections.bias: 'bias'
+      connections.dark: 'dark'
+      connections.flat: 'flat'
+      doWrite: true
+      doOverscan: true
+      doAssembleCcd: true
+      doBias: true
+      doVariance: true
+      doLinearize: parameters.linearize
+      doCrosstalk: parameters.crosstalk
+      doBrighterFatter: false
+      doDark: true
+      doStrayLight: false
+      doFlat: false
+      doFringe: false
+      doApplyGains: false
+      doDefect: parameters.defects
+      doNanMasking: true
+      doInterpolate: false
+      doSaturation: false
+      doSaturationInterpolation: false
+      growSaturationFootprintSize: 0
+  cptPtcExtract:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveExtractTask
+    config:
+      connections.inputExp: 'cptPtcProc'
+      connections.taskMetadata: 'cptPtcIsr_metadata'
+      connections.outputCovariances: 'cptPtcExtract'
+      matchExposuresType: "EXPID"
+  cptPtcSolve:
+    class: lsst.cp.pipe.ptc.PhotonTransferCurveSolveTask
+    config:
+      connections.inputCovariances: 'cptPtcExtract'
+      connections.outputPtcDataset: 'ptc'
+      ptcFitType: EXPAPPROXIMATION
+
+  #linearity
+  cptLinearitySolve:
+    class: lsst.cp.testing.CptLinearitySolveTask
+    config:
```

Comment:
Should this have the configs using the photodiode as an input as well?

Reply:
Probably.  I've added it in, and will check that tests do not fail.